packages/cactus-plugin-ledger-connector-fabric/src/main/typescript/prometheus-exporter/Readme.md

```diff
@@ -0,0 +1,39 @@
+# Prometheus Exporter
```

Comment:
@jagpreetsinghsasan Please move the content from this file to the `README.md` file in the package's root directory and increase the header levels by one in each section (e.g. `#` becomes `##`, `##` becomes `###` etc..)
This file itself then can be deleted so that in the src/main/typescript directory with don't end up having documentation that's not typescript code.

Reply:
Fixed